specification/paths/Brokers-broker_id-carriers.json

```diff
@@ -0,0 +1,109 @@
+                "data"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "allOf": [
+                      {
+                        "$ref": "#/components/schemas/Carrier"
+                      },
+                      {
+                        "required": [
+                          "id",
+                          "links"
+                        ],
+                        "properties": {
+                          "attributes": {
+                            "required": [
+                              "status",
+                              "currency",
+                              "name",
+                              "created_at"
+                            ]
+                          },
+                          "relationships": {
+                            "properties": {
+                              "owner": {
+                                "example": {
+                                  "links": {
+                                    "related": "$API_HOST/carriers/9cdf86e8-333f-4ed9-bb31-4935c780c947"
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    ]
+                  }
+                },
+                "meta": {
+                  "$ref": "#/components/schemas/PaginationMeta"
+                },
+                "links": {
+                  "type": "object",
+                  "required": [
+                    "self",
+                    "first",
+                    "last"
+                  ],
+                  "properties": {
+                    "self": {
+                      "type": "string",
+                      "example": "API_HOST/organizations/9cdf86e8-333f-4ed9-bb31-4935c780c947/contracts?page[number]=3&page[size]=30"
+                    },
+                    "first": {
+                      "type": "string",
+                      "example": "API_HOST/organizations/9cdf86e8-333f-4ed9-bb31-4935c780c947/contracts?page[number]=1&page[size]=30"
+                    },
+                    "prev": {
+                      "type": "string",
+                      "example": "API_HOST/organizations/9cdf86e8-333f-4ed9-bb31-4935c780c947/contracts?page[number]=2&page[size]=30"
+                    },
+                    "next": {
+                      "type": "string",
+                      "example": "API_HOST/organizations/9cdf86e8-333f-4ed9-bb31-4935c780c947/contracts?page[number]=4&page[size]=30"
+                    },
+                    "last": {
+                      "type": "string",
+                      "example": "API_HOST/organizations/9cdf86e8-333f-4ed9-bb31-4935c780c947/contracts?page[number]=13&page[size]=30"
```

Comment:
All the links above should be replaced with `/brokers/a294ee55-bc94-4890-b734-afd56c158f95/carriers`

Reply:
Ah so these 'examples' reference actual existing entities?

Reply:
oh now I see I haven't renamed the example organizations into brokers. My bad.

Reply:
Yeah that was the main thing :) The UUID values are not referencing anything, but we aim to use the same UUID when we talk about the same resource (and have each resource have a different one).